calyx-opt/src/traversal/action.rs

```diff
@@ -30,7 +35,10 @@ impl Action {
     {
         match self {
             Action::Continue => next(),
-            Action::Change(_) | Action::Stop | Action::SkipChildren => Ok(self),
+            Action::Change(_)
```

Comment:
Maybe `Change` should use a `GenericControl` operator so we don't need a separate `StaticChange`. Not sure if this is a good idea